### ISSUE TITLE
sfdc must mot lowercase basename

### ISSUE THREAD

**Issue by sezero** [adtools/sfdc#3]
At present, sfdc lowercases basename, and that results in broken proto/Warp3D.h
with `#include <clib/warp3d_protos.h>` instead of `#include <clib/Warp3D_protos.h>`

For a similar past issue, see: https://github.com/adtools/fd2pragma/commit/a640e4a69e4f11dd648eb7cd84938c9b46ccef62 

I suggest something like below: (minimally tested)
```
diff --git a/sfdc b/sfdc
index 3b8cb2d..f35e7af 100755
--- a/sfdc
+++ b/sfdc
@@ -689,9 +689,9 @@ sub parse_sfd ( $ ) {
     }
 
     $$result{'basename'} =~ s/-/_/g;
-    $$result{'basename'} = lc $$result{'basename'};
     $$result{'BASENAME'} = uc $$result{'basename'};
-    $$result{'Basename'} = ucfirst $$result{'basename'};
+    $$result{'Basename'} = lc $$result{'basename'};
+    $$result{'Basename'} = ucfirst $$result{'Basename'};
     ($result->{BaseName} = $result->{base}) =~ s/Base//;
 
     return $result;

```

**Comment by jens-maus** [comment]
Thanks for your contribution. Please provide a proper pull request so that I can accept this change ASAP.

**Comment by jens-maus** [comment]
Implemented by d2dc947a469bd8ee72228b37d03d2c921d2b0b38
